iconsdk/utils/__init__.py

```diff
@@ -40,16 +42,21 @@ def inner(*args, **kwargs):
             return callback(fn(*args, **kwargs))
 
         return inner
+
     return outer
 
 
 to_dict = apply_to_return_value(dict)
 
 
-def set_logger(logger, level):
-    handler = StreamHandler()
-    formatter = Formatter(
-        '%(asctime)s %(name)-12s %(levelname)-8s %(message)s')
+def set_logger(level, handler=StreamHandler()):
```

Comment:
How about adding a 'format' parameter?

ICON-SDK-PYTHON users may want their own log format.

Reply:
Modified code for users to use their own log format.
In addition, add more options on default format. 

AS-IS
%(asctime)s %(name)-12s %(levelname)-8s %(message)s

TO-BE
%(asctime)s %(name)-12s %(levelname)-5s %(filename)-12s %(lineno)-4s %(funcName)-12s %(message)s